Makefile

```diff
@@ -203,10 +208,31 @@ generate-fakes: ## Generate fakes for writing unit tests
 	$(GO) generate ./...
 	$(MAKE) fmt
 
+.PHONY: e2e-cli-core
+e2e-cli-core: fmt vet ## Run CLI Core E2E Tests
```

Comment:
Does this target always need me to start the local registry?  If so, do you think it would be good to add it as a dependency target?  And we could call `$(MAKE) clean-registry` after the tests are run?